lite-rpc/src/bridge.rs

```diff
@@ -440,8 +494,42 @@ impl LiteRpcServer for LiteBridge {
         todo!()
     }
 
-    async fn slot_subscribe(&self, _pending: PendingSubscriptionSink) -> SubscriptionResult {
-        todo!()
+    async fn slot_subscribe(&self, pending: PendingSubscriptionSink) -> SubscriptionResult {
+        let sink = pending.accept().await?;
+        let mut slot_stream = self.slot_stream.resubscribe();
+        tokio::spawn(async move {
```

Comment:
Doesn't look good. Any one will break down lite-rpc with a lot of empty requests. Preferred moving this to a `Semaphore` implementation using channels. 

1. Maintain one `slot_stream`.
2. Poll if there is a slot_subscribe request.
3. Respond to all requests in a single loop.
4. Sleep for a while if no requests come in or wait on the channel.

Prevents spawning multiple tokio::spawn

Reply:
Tokio is made to have multiple tasks, I guess should fine like this instead of making something more complex.

Reply:
Tokio.rs states 
```
Tasks in Tokio are very lightweight. Under the hood, they require only a single allocation and 64 bytes of memory. Applications should feel free to spawn thousands, if not millions of tasks.
```

Reply:
I don't agree. Previous experiences say this will end badly.